nixos/modules/services/web-apps/lemmy.nix

```diff
@@ -47,10 +69,6 @@ in
           description = lib.mdDoc "Port where lemmy should listen for incoming requests.";
         };
 
-        options.federation = {
-          enabled = mkEnableOption (lib.mdDoc "activitypub federation");
-        };
-
```

Comment:
I tried to add a `mkRemovedOptionModule` for this but it caused evals to fail. Is there something else I need to use for this, or is it okay to just remove the option?

Reply:
Also maybe I should put the `remove option removed upstream` commit in a separate PR that won't get backported?

Reply:
Marking this as resolved since it was changed to issue a warning.